sidecars/java/etc/entrypoint.sh

```diff
@@ -27,4 +27,9 @@ if [ "${USER_ID}" -ne 0 ] && command -v sudo >/dev/null 2>&1 && sudo -n true > /
     sudo chown "${USER_ID}:${GROUP_ID}" /projects
 fi
 
+echo 'Setting "java.semanticHighlighting.enabled" to false'
```

Comment:
how does it work with multi-root workspaces ? 
I think these kind of settings may be ignored
cc @RomanNikitenko

Reply:
How will multi-root workspaces deal with "workspace" settings ? like the one defined in the devfile ?

Reply:
https://github.com/eclipse/che/issues/17975#issuecomment-760229209 and the follow-up comment. Answer: we need some new code.

Reply:
But same is applying for the settings you apply in the devfile like: https://github.com/eclipse/che-devfile-registry/blob/dcfdacf3954e30a25a8af3c7bd497b193d24f14a/devfiles/quarkus/devfile.yaml#L59-L62
So the strategy needs to be covered in the multi-root PR

Reply:
I posted my thoughts about it here https://github.com/eclipse/che/issues/17212#issuecomment-778575536
Please share your opinion